app/assets/stylesheets/modules/edit_form.scss

```diff
@@ -0,0 +1,40 @@
+@import "variables";
+@import "mixins";
+
+.form {
+
+  .legend {
+    font-family: $dogstay-font-theme;
+    font-size: 2em;
+  }
+
+  img {
+    @include image-style;
+    height: 15em;
+    margin-left: 0;
+    margin-right: 0;
+    margin-top: 3em;
+    width: 15em;
+  }
+
+  ul {
+    font-family: $sitter-font-type;
+    padding-left: 2.2em;
+  }
+
+  .postfix {
+    background: #008080;
```

Comment:
Color literals like `#008080` should only be used in variable declarations; they should be referred to via variable everywhere else.